resources/lang/de/Config.properties

```diff
@@ -227,6 +206,32 @@ basketPreviewTypeLabel = Art der Warenkorbvorschau
 basketPreviewTypeHover = Dropdown bei Mouse-over
 basketPreviewTypeRight = Rechts einblenden
 
+contactTab=Kontaktformular
+contactShopMailLabel = E-Mail-Adresse des Kontaktformulars
+contactShopMailCcLabel = Zusätzliche Cc E-Mail-Adressen für das Kontaktformular (Mehrere getrennt durch ein Komma)
```

Comment:
E-Mail-Adresse für Kopie der Kontaktanfrage (CC)


Diese Info in Klammern ergänzen wir im Handbuch (mehrere Adressen kommasepariert eingeben)